scripts/sync-and-test-bucket.sh

```diff
@@ -40,7 +40,8 @@ destination_bucket_uri="s3://${destination_bucket}"
 node scripts/translate-redirects.js "$build_dir" "$(pulumi -C infrastructure config get redirectDomain 2>/dev/null || echo '')"
 
 # Read the region from the stack's config -- we use it below.
-aws_region="$(pulumi -C infrastructure config get 'aws:region')"
+aws_region="$(pulumi -C infrastructure config get 'aws:region')" || aws_region="us-west-2"
```

Comment:
This is so that PR previews can still be deployed without an actual stack being created, which is where we currently get the region from.

Reply:
Might suggest a small tweak to define the variable once:

```suggestion
aws_region="$(pulumi -C infrastructure config get 'aws:region' 2>/dev/null || echo 'us-west-2')"
```

.. but more importantly, why would someone want to do this in practice?

Reply:
> but more importantly, why would someone want to do this in practice?

I could fork this repo and not create a stack and still get my PR preview builds working.